styles/sections/_schedule.scss

```diff
@@ -0,0 +1,103 @@
+@use '../colors';
+
+.schedule-list {
+  display: flex;
+  flex-direction: column;
+  text-transform: capitalize;
+  padding-top: 8px;
+  position: relative;
+
+  @container body (min-width: 1024px) {
+    &:before {
```

Comment:
There's some weirdness around the lines to the left and right of the schedule bubbles, esp. between 1024px and 1439px. Is there another way we can add those in? Maybe they can grow in from the outer edges of the browser inward, or just automatically show up when the user hits 1024px wide then always show from that size and up?

- ![Screenshot 2023-04-17 at 8 22 58 AM](https://user-images.githubusercontent.com/60403038/232534886-10064195-7379-4e5d-be14-5664963d0f1c.png)

- ![Screenshot 2023-04-17 at 8 22 05 AM](https://user-images.githubusercontent.com/60403038/232534894-c05148d0-061c-47a9-8046-dac86ee6ac39.png)

- ![Screenshot 2023-04-17 at 8 21 53 AM](https://user-images.githubusercontent.com/60403038/232534898-fa802d3e-ab31-4202-bfce-3cc6f5f8d894.png)

Reply:
Yeah, I was having issues getting the lines working on smaller desktop sizes on Friday 😓 I re-evaluated today and realized it was because the lines are absolutely positioned and the schedule section isn't always a fixed width, so I added some styles today to make it less flex-y on the smaller desktop sizes. I just pushed those fixes up :-)